packages/cli/src/commands/compare/index.ts

```diff
@@ -144,13 +148,33 @@ export default class Compare extends Command {
             `Could not sample from provided urls\nCONTROL: ${this.parsedConfig.controlURL}\nEXPERIMENT: ${this.parsedConfig.experimentURL}.`
           );
         }
+        const resultJSONPath = `${this.parsedConfig.tbResultsFolder}/compare.json`;
 
-        fs.writeFileSync(
-          `${this.parsedConfig.tbResultsFolder}/compare.json`,
-          JSON.stringify(results, null, 2)
-        );
+        fs.writeFileSync(resultJSONPath, JSON.stringify(results, null, 2));
+        // tslint:disable-next-line: max-line-length
+        const message = `${chalk.black.bgGreen(' Success! ')} ${fidelity} test samples were taken. The JSON file with results from the compare test are available here: ${tbResultsFolder}/compare.json.`;
```

Comment:
Include the `chalk.black.bgGreen` into the `chalkScheme` https://github.com/TracerBench/tracerbench/blob/master/packages/cli/src/helpers/utils.ts#L214-L230